docs/reference/search/async-search.asciidoc

```diff
@@ -80,12 +80,20 @@ results are returned as part of the <<search-api-response-body,`response`>> obje
 // TESTRESPONSE[s/"relation" : "gte"/"relation": $body.response.hits.total.relation/]
 // TESTRESPONSE[s/"hits" : \[ \]\n\s\s\s\s\}/"hits" : \[\]},"aggregations":  $body.response.aggregations/]
 
-<1> Identifier of the async search that can be used to monitor its progress, retrieve its results, and/or delete it.
-<2> Whether the returned search results are partial or final
+<1> Identifier of the async search that can be used to monitor its progress,
+retrieve its results, and/or delete it.
+<2> Whether the returned search results are coming from all the shards that
+the query was expected to hit
 <3> Whether the search is still being executed or it has completed
```

Comment:
should we add `completed or failed` ?

Reply:
Something like "whether all the shards that the query targets either completed or failed". ?

Reply:
I was more thinking of fatal failures so no mention of shards imo.

Reply:
I rephrased it, I think this makes the naming of this parameter quite off though :) it could rather be `failed`

Reply:
reading again this explanation, I am no longer sure that I like it. It is perfectly fine to have partial responses while the query is running. The only case where `is_partial` set to `true` indicates that something has failed is when the query is no longer running (`is_running` set to `false`). I will rephrase, also I believe we should remove is_partial based on the conclusions I came to in #55572 .